Configurations.md

```diff
@@ -1248,12 +1248,20 @@ Control the case of the letters in hexadecimal literal values
 
 ## `hide_parse_errors`
 
-Do not show parse errors if the parser failed to parse files.
+This option is deprecated. Use `show_parse_errors` instead. 
 
 - **Default value**: `false`
 - **Possible values**: `true`, `false`
 - **Stable**: No (tracking issue: [#3390](https://github.com/rust-lang/rustfmt/issues/3390))
 
+## `show_parse_errors`
+
+Show parse errors if the parser failed to parse files.
+
+- **Default value**: `true`
+- **Possible values**: `true`, `false`
+- **Stable**: Yes
```

Comment:
`hide_parse_errors` was an unstable option. This also needs to be an unstable option.

Reply:
What tracing issue should be mentioned? The same as the unstable option ([#3390](https://github.com/rust-lang/rustfmt/issues/3390))? Unstable option's issue is about renaming it, so logically that issue is resolved in this PR.

Reply:
Feel free to create a new tracking issue (I can add the `unstable opiton` label). We can also leave the tracking issue blank for now and follow up with it in a future PR. Up to you!